third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -561,25 +632,46 @@ func resourceSqlDatabaseInstanceCreate(d *schema.ResourceData, meta interface{})
 	d.Set("name", name)
 
 	instance := &sqladmin.DatabaseInstance{
+		<% if version == 'ga' -%>
 		Name:                 name,
 		Region:               region,
 		Settings:             expandSqlDatabaseInstanceSettings(d.Get("settings").([]interface{}), !isFirstGen(d)),
 		DatabaseVersion:      d.Get("database_version").(string),
 		MasterInstanceName:   d.Get("master_instance_name").(string),
 		ReplicaConfiguration: expandReplicaConfiguration(d.Get("replica_configuration").([]interface{})),
+		<% else %>
+		Name:            name,
+		Region:          region,
+		DatabaseVersion: d.Get("database_version").(string),
+		<% end -%>
 	}
 
 	<% unless version == 'ga' -%>
 	// MSSQL Server require rootPassword to be set
 	if strings.Contains(instance.DatabaseVersion, "SQLSERVER") {
 		instance.RootPassword = d.Get("root_password").(string)
 	}
+
+	if sqlDatabaseIsOnPremises(d) {
+		instance.OnPremisesConfiguration = &sqladmin.OnPremisesConfiguration{
+			HostPort:        d.Get("source_ip_address").(string) + ":" + d.Get("source_port").(string),
+			Kind:            "sql#onPremisesConfiguration",
```

Comment:
`Kind` usually doesn't actually need to be set

Reply:
Removed